tests/Mono.Android-Tests/Mono.Android-Tests.csproj

```diff
@@ -21,7 +21,7 @@
     <MandroidI18n>All</MandroidI18n>
     <SignAssembly>true</SignAssembly>
     <AssemblyOriginatorKeyFile>..\..\product.snk</AssemblyOriginatorKeyFile>
-    <TargetFrameworkVersion>v12.0</TargetFrameworkVersion>
+    <TargetFrameworkVersion>v12.1</TargetFrameworkVersion>
```

Comment:
Should this (and the two projects above) be using `$(AndroidLatestStableFrameworkVersion)` instead?